src/main/java/csv/ReadPersonCsvDemo.java

```diff
@@ -0,0 +1,32 @@
+package main.java.csv;
+
+import main.java.serialization.Person;
+import main.java.serialization.PersonService;
+
+import java.util.List;
+
+public class ReadPersonCsvDemo {
+    private final static String PEOPLE_DATA = "people.csv";
```

Comment:
Reorder the modifiers to comply with the Java Language Specification.

Correct order: `static final`